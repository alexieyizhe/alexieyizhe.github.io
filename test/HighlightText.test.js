import React from 'react';
import HighlightText from '../src/components/HighlightText';


describe('Highlight Text', () => {
  let text;

  global.___loader = {
    enqueue: jest.fn()
  };

  beforeEach(() => {
    text = render(
      <HighlightText color="#F8BJH3" hovered={false}>
        "Test text"
      </HighlightText>
    ).children();
  });

  describe('default state', () => {
    it('should be defined', () => {
      expect(text).toBeDefined();
    });
  });

  it('should render the highlight text', () => {
    expect(text).toMatchSnapshot();
  });

});
