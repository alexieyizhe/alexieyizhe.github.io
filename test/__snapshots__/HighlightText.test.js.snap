// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Highlight Text should render the highlight text 1`] = `null`;
